### PR TITLE
Fix adding named route

### LINKED ISSUE #23: ValueError: Duplicate 'handler', already handled by <DynamicRoute 'handler' [GET] /v1/handler/ -> <function handler at 0x103c85378>
I get a `ValueError` when passing a name to `router.add_route`

```python
        cors.add(
            app.router.add_route(method, full_url, handler, name=name),
            {
                '*': aiohttp_cors.ResourceOptions(
                    allow_credentials=True, expose_headers='*', allow_headers='*'
                )
            }

        )
```

Traceback:

```
    app = create_app(TestConfig)
api/app.py:27: in create_app
    routes.setup(app)
api/routes.py:26: in setup
    allow_credentials=True, expose_headers='*', allow_headers='*'
../../../miniconda/envs/rero/lib/python3.5/site-packages/aiohttp_cors/__init__.py:193: in add
    route, self._preflight_handler)
../../../miniconda/envs/rero/lib/python3.5/site-packages/aiohttp_cors/urldispatcher_router_adapter.py:45: in add_options_method_handler
    self._router.register_route(new_route)
../../../miniconda/envs/rero/lib/python3.5/site-packages/aiohttp/web_urldispatcher.py:459: in register_route
    .format(name, self._routes[name]))
E   ValueError: Duplicate 'should_i_release', already handled by <DynamicRoute 'should_i_release' [GET] /v1/should_i_release/{username}/{repo}/ -> <function should_i_release at 0x103c85378>
```

Removing `name=name` from the call to `app.router.add_route` removes the error.

### PR DESCRIPTION
**Pull request by rutsky**
Fixes #23 .

### CODE REVIEW

File: aiohttp_cors/urldispatcher_router_adapter.py [hunk 1]
```diff
@@ -35,13 +35,14 @@ def route_methods(self, route: web.Route):
 
     def add_options_method_handler(self, route: web.Route, handler):
         method = "OPTIONS"
+        name = None
```
Comment (asvetlov) [inline aiohttp_cors/urldispatcher_router_adapter.py]: Thus you add anonymous (unnamed) route for OPTIONS method, right?
Please don't use `name=None` variable for this but pass `None` explicitly.

Comment (rutsky) [inline aiohttp_cors/urldispatcher_router_adapter.py]: Done.

Comment (rutsky) [inline aiohttp_cors/urldispatcher_router_adapter.py]: > Thus you add anonymous (unnamed) route for OPTIONS method, right?

Yes, I'm adding unnamed route for OPTIONS method for the same path, as `route` has.

**Comment by asvetlov** [comment]
LGTM.

**Pull request merged**